proofread:postgresql_best_practices.md (#2385)
juicedata/juicefs: docs/en/administration/metadata/postgresql_best_practices.md

## Authentication methods

> The "md5" method is currently available for authentication with PostgreSQL. The following section can be adapted in the pg_hba.conf of your PostgreSQL instance.

Reviewer: supported

> The "md5" method is currently available for authentication with PostgreSQL. The following section can be adapted in the pg_hba.conf of your PostgreSQL instance.

Author: ```suggestion
PostgreSQL supports the md5 authentication method. The following section can be adapted in the pg_hba.conf of your PostgreSQL instance.
```
